### DISCUSSION TITLE
Windows 10 x64 UI version

### DISCUSSION THREAD

**Discussion by bandaid** [liquidctl/liquidctl#290] [Ideas]
I am an idiot. Can we get a simple UI version of liquidctl? Also, does this control both the fan and pump speeds?

**Comment by jonasmalacofilho** [comment]
There are a few people working on GUIs for liquidctl, but I don't whether they plan to support Windows or your device.

[GKraken](https://gitlab.com/leinardi/gkraken/activity) is probably the most usable right now, and may already support Windows.  On the other hand, it definitively only supports fan/pump *speed* control on third and fourth generation Kraken X/Z coolers.

You can also follow updates on #188 (and the repository linked in that thread).
